src/main/java/org/takes/facets/auth/social/PsGithub.java

```diff
@@ -100,10 +125,9 @@ public Response exit(final Response response,
      * @return The user found in Github
      * @throws IOException If fails
      */
-    private static Identity fetch(final String token) throws IOException {
-        final String uri = new Href("https://api.github.com/user")
-            .with("access_token", token)
-            .toString();
+    private Identity fetch(final String token) throws IOException {
+        final String uri = new Href(String.format("%s/user", this.api))
```

Comment:
this is not really a safe way to build URLs. I would rather encapsulate an instance of `Href` (instead of `String`) and do something like this here: `this.api.path("user").with("access_token", token)`